test/test_torch.py

```diff
@@ -4629,24 +4629,66 @@ def test(name):
     def test_signal_window_functions(self):
         self._test_signal_window_functions(self)
 
-    @skipIfNoLapack
-    def test_inverse(self):
-        M = torch.randn(5, 5)
+    @staticmethod
+    def _test_inverse(self, cast):
+        from common import random_fullrank_matrix_distinct_singular_value
+
+        # no batches: 2-D tensors
+        M = cast(random_fullrank_matrix_distinct_singular_value(5))
```

Comment:
nit: while we're touching this test anyways, it would be nice if these were not one-letter names "matrix, matrix_inverse, expected" would be great